index.html

```diff
@@ -274,6 +274,229 @@ <h3>
+        <aside class="example">
+          <p>
+            In the following example, the developer has included two
+            <a>protocol handler description</a> <var>protocol_handler</var>s.
+            Assuming the the manifest's URL is
+            <samp>https://example.com/manifest.webmanifest</samp>:
+          </p>
+          <ul>
+            <li>The first protocol handler would register to handle "web+music"
+            URLs (e.g.: web+music://#1234). When activated, the user agent
+            would instantiate a new <a>top-level browsing context</a> and
+            navigate to
+            <samp>https://example.com/play?songId=web+music://%231234</samp>.
+            </li>
+            <li>The second protocol handler would be ignored, as the protocol
+            provided does not start with "web+" and is not part of the
+            [=safelisted schemes=].
+            </li>
+          </ul>
+          <pre class="example json">
+            {
+              "protocol_handlers": [
+                {
+                  "protocol": "web+music",
+                  "url": "/play?songId=%s"
+                },
+                {
+                  "protocol": "store",
+                  "url": "/buy?songId=%s"
+                }
+              ]
+            }
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h2>
+          ProtocolHandler items
+        </h2>
+        <p>
+          Each <dfn>protocol handler description</dfn> is an [=object=] that
+          represents a protocol that the web application wants to handle. It
+          has the following members:
+        </p>
+        <ul>
+          <li>[=`protocol`=]
+          </li>
+          <li>[=`url`=]
+          </li>
+        </ul>
+        <p>
+          A user agent SHOULD use these values to register the web application
+          as a handler with the operating system. When the user activates a
+          protocol handler URL, the user agent SHOULD run <a>handling a
+          protocol launch</a>.
+        </p>
+        <p class="note">
+          [[HTML]]'s {{NavigatorContentUtils/registerProtocolHandler()}} allows
+          web sites to register themselves as possible handlers for particular
+          protocols. What constitutes valid <code>protocol</code> and
+          <code>url</code> values for <a>protocol handler description</a>s is
+          defined in that API. Also note that the [[HTML]] API uses
+          [=url/scheme=] where we use <code>protocol</code> but the same
+          restrictions apply.
+        </p>
+        <section>
+          <h3>
+            `protocol` member
+          </h3>
+          <p>
+            The <dfn>protocol</dfn> member of a <a>protocol handler
+            description</a> is a <a>string</a> that represents the protocol to
+            be handled, such as `mailto` or `web+auth`.
+          </p>
+          <p>
+            The <a>protocol</a> member of a <a>protocol handler description</a>
```

Comment:
all of these <a> links should be [=this style=] links I believe - unless they are referencing an IDL thing, in which case they should be {{tthis/style}}.

See https://respec.org/docs/#definitions-and-linking